### PR TITLE
Improve MooseDocs docs

### LINKED ISSUE #11948: Add MOOSE_DIR to moosedocs instructions
## Rationale
Incorrect (incomplete) instructions for:
http://mooseframework.org/moose/utilities/MooseDocs/setup.html

## Description
When using stork.sh to create an app, and then following these instructions:
  http://mooseframework.org/moose/utilities/MooseDocs/setup.html
The documentation system does not work

## Impact
None

### PR DESCRIPTION
**Pull request by aeslaughter**
closes #11948

### CODE REVIEW

**Comment by aeslaughter** [comment]
@brianmoose Can you add MooseDocs testing to the stork target, please.

File: framework/doc/content/utilities/MooseDocs/config.md [hunk 1]
```diff
@@ -1,5 +1,107 @@
...
+directory below the content directory, regardless of depth.
+
+### Advanced Content
+
+There is also a more advanced interface for the content, but it is required to explain how these
+directories are used. When building content all relevant files (e.g., markdown) are extracted from
+the supplied list of directories into a set, with the path relative to the supplied directory. This
+unique set of files is then copied to the destination directory of the build, with markdown files
+being rendered to html first. For example, the following illustrates how the multiple source
+files are combined in the destination directory (
```
Comment (brianmoose) [inline framework/doc/content/utilities/MooseDocs/config.md]: weird (

File: framework/doc/content/utilities/MooseDocs/setup.md [hunk 2]
```diff
@@ -1,48 +1,65 @@
...
+## Dependencies
 
-To use MooseDocs, an executable is required---this main executable is simply copied from the
-executable within [MOOSE]:
+If you are not using a MOOSE package or your package is old, then the following packages must be
+installed, which can be done using [pip](https://pip.pypa.io/en/stable/).
 
 ```bash
-cp ~/projects/moose/modules/doc/moosedocs.py ~/projects/your_application_name/doc
+pip install --user pybtex livereload pylatexenc anytree pandas
```
Comment (brianmoose) [inline framework/doc/content/utilities/MooseDocs/setup.md]: Do we need to use `livereload==2.5.1`?

**Review by brianmoose — CHANGES_REQUESTED** [review]

**Review by brianmoose — APPROVED** [review]

**Comment by dschwen** [comment]
A cool improvement would also be to have a working `./moosedocs.py check --generate` (#11375)... :-)

**Pull request merged**